savoirfairelinux/num2words
# Fixes lang_DK issues

## Fixes lang_DK issues

### Changes proposed in this pull request:

* Fixed GIGA_SUFFIX typo in `lang_DK.py´
* Set `self.ordflag` to `False` by default in `lang_DK.py´

### Status


### How to verify this change

Run the following piece of code and observe that it runs without issues.
```python
import num2words
print(num2words.num2words(1000000000, lang='dk'))
```

### Additional notes

No additional notes.
